iphone/Classes/TiUITableView.m

```diff
@@ -139,7 +142,10 @@ -(void)touchesEnded:(NSSet *)touches withEvent:(UIEvent *)event
         || ([[event touchesForView:self.imageView] count] > 0) || ([[event touchesForView:self.proxy.currentRowContainerView] count]> 0 )) {
         if ([proxy _hasListeners:@"touchend"])
         {
-            [proxy fireEvent:@"touchend" withObject:[proxy createEventObject:nil] propagate:YES];
+            UITouch *touch = [touches anyObject];
+            CGPoint point = [touch locationInView:self];
+            NSDictionary *evt = [TiUtils touchPropertysToDictionary:touch:[touch locationInView:self]];
+            [proxy fireEvent:@"touchstart" withObject:evt propagate:YES];
```

Comment:
Do not copy and paste code. In this example, the "touchstart" event would be fired if the "touchend" listener is added.